evadb/database.py

```diff
@@ -39,7 +44,14 @@ def catalog(self) -> "CatalogManager":
 
 
 def get_default_db_uri(evadb_dir: Path):
-    return f"sqlite:///{evadb_dir.resolve()}/{DB_DEFAULT_NAME}"
+    import yaml
+
+    f = open(Path(EvaDB_INSTALLATION_DIR) / "evadb.yml", "r+")
+    config_obj = yaml.load(f, Loader=yaml.FullLoader)
+    if config_obj["experimental"]["use_postgres_backend"] is False:
+        return f"sqlite:///{evadb_dir.resolve()}/{DB_DEFAULT_NAME}"
+    else:
+        return f"postgresql://postgres:password@localhost:5432/{PG_DB_DEFAULT_NAME}"
```

Comment:
Same as above, we should allow users to specify an existing Postgres db server.

Reply:
Done